tools/distrib/buildifier_format_code.sh

```diff
@@ -1,4 +1,4 @@
-#! /bin/bash -ex
+#! /bin/bash -e
```

Comment:
I think using `set -e` at the beginning of the script is much more readable. (No one looks above the license header).
Also below.